roseus/test/param-test.l

```diff
@@ -173,6 +173,24 @@
              (bench2 (dotimes (i 100) (ros::get-param "test"))))
           "(ros::get-param-cached test) has less performance than (ros::get-param test)"))
 
+(deftest test-list-param ()
+  (let (ret)
+    (assert (not (set-difference (cdr ret)
+                                 '("/test_dictionary/str_value" "/test_dictionary/dictionary/key_bool" "/test_dictionary/dictionary/key_dbl" "/test_dictionary/dictionary/key_str" "/test_dictionary/dictionary/key_int" "/test_dictionary/dictionary/key_list" "/test_dictionary/list" "/test_dictionary/bool_value" "/test_dictionary/int_value" "/test_dictionary/dbl_value" "/rosversion" "/run_id" "/param/test" "/param/read_parameter/test" "/test" "/rosdistro")
+                                 :test #'equal))
+            (format nil "list-param ~A failed" ret))
+    ))
+
+(deftest test-search-param ()
+  (let (ret)
+    (ros::roseus "param_test") ;; this would be read_parameter in test-parameter.test
```

Comment:
It is still `"read_parameter"`, since calling `ros::roseus` again does not overwrite the installed name

Reply:
Yes it is expected to remap to "read_parameter", so that
```
    (setq ret (ros::search-param "test"))
    (assert (string= ret "/param/read_parameter/test") "search-param returns ~A" ret)
```
passes

Reply:
Do we need the `(ros::roseus "param_test")` line in here? 
The assertation should work fine without it.